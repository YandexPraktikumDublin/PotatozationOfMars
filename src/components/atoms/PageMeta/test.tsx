import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { PageMeta } from '.'

describe('<PageMeta />', () => {
  it('should renders correct <PageMeta />', () => {
    const wrapper = shallow(<PageMeta />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
