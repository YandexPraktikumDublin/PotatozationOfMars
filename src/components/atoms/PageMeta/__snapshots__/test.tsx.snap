// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageMeta /> should renders correct <PageMeta /> 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <title>
    Potatozation of Mars
  </title>
  <meta
    content="The main character is a businessman Elon Musk, who is flying to Mars for the purpose of colonization, the cargo of his spaceship is potatoes, the cultivation of which he wants to establish in a future colony. After losing all his cargo, he returns to Earth to try again and the game starts again."
    name="description"
  />
</HelmetWrapper>
`;
